src/tests/Microsoft.PowerFx.Interpreter.Tests/TimeTests.cs

```diff
@@ -54,7 +54,7 @@ public void TestCurrentTimeZoneOffset()
         public void TestConvertToUTC()
         {
             var testTime = DateTime.Now;
-            var result = _engine.Eval($"DateTimeValue(\"{testTime.ToString()}\") + TimeZoneOffset()");
+            var result = _engine.Eval($"DateAdd(DateTimeValue(\"{testTime.ToString()}\"), TimeZoneOffset(), \"Minutes\")");
```

Comment:
>\"Minutes\" [](http://example.com/codeflow?start=108&length=11)

[minor] Let's use the enum value (`TimeUnit.Minutes`) instead of its internal representation - that's a better example that we can give to users